azure-functions/create-function-app-connect-to-storage/create-function-app-connect-to-storage-account.sh

```diff
@@ -6,3 +6,6 @@ az storage account create --name myfunctionappstorage --location westeurope --re
 
 # create a new function app, assign it to the resource group you have just created
 az functionapp create --name myFunction --resource-group myResourceGroup --storage-account myfunctionappstorage --consumption-plan-location westeurope
+
+# update function app settings to connect to storage account
+az functionapp config appsettings update --name MyFunction --resource-group myResourceGroup --settings setting=value
```

Comment:
We should get the actual Storage Connection String and store it in the App Setting. See this example:  https://github.com/Azure/azure-docs-cli-python-samples/blob/master/app-service/connect-to-storage/connect-to-storage.sh